### ISSUE TITLE
Fails to parse \@writefile{lot}{\egroup }

### ISSUE THREAD

**Issue by no-preserve-root** [nlct/texparser#2]
Since v1.45, polyglossia emits the line
```\@writefile{lot}{\egroup }```
into the aux file if language switches occur in the document. This causes the parser to fail and makes `bib2gls` unusable.

Speculation: the parser does not understand that `\egroup` must not be processed in `\@writefile`?

### MWE

```
% test.tex
\documentclass{article}

\usepackage{polyglossia}
\setdefaultlanguage{german}

\usepackage[record=only]{glossaries-extra}
\GlsXtrLoadResources[src={glossary}]

\begin{document}
\begin{german}
\end{german}
\gls{gls:abc}
\printunsrtglossaries
\end{document}
```

```
%glossary.bib
@acronym{gls:abc,
    short = {ABC},
    long  = {Aa Bb Cc}
}
```

Create a `glossary.bib` with an entry for `gls:abc`.

### Expected Result

Document compiles.

### Actual Result

```
$ bib2gls test
bib2gls version 2.0 (2020-02-12)
Reading test.aux
Error: test.aux:36: Unexpected end group found
Reading glossary.bib
Writing /tmp/test/test.glstex
1 entry written to /tmp/test/test.glstex.
Transcript written to test.glg.
```

**Comment by nlct** [comment]
I think the simplest thing to do is for the parser to define `\@writefile` to ignore its arguments. At the moment it's ignoring `\@writefile` because it's unknown so the parser sees `\egroup` out of context.

**Comment by no-preserve-root** [comment]
@nlct do you intend to fix this in the near future?

**Comment by nlct** [comment]
It's turning out to be more complicated than I had originally thought. I don't know how long it will take to fix.

**Comment by nlct** [comment]
Now fixed in `texparserlib.jar` v0.9.1b, which is included in the new `bib2gls` v2.1 (2020-02-28) release. I've uploaded the new version of `bib2gls` to CTAN. It may take a few days to make it into the TeX distributions.
